src/auth/verifyingPopUpLoginSASVIYA.ts

```diff
@@ -0,0 +1,29 @@
+import { delay } from '../utils'
+
+export async function verifyingPopUpLoginSASVIYA(loginPopup: Window) {
```

Comment:
I'd call this file and function `verifySasViyaLoginPopup`.

Reply:
fixed.

Reply:
fixed.